enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -859,7 +859,7 @@ func (l *FileCacheLoader) cacheCOW(ctx context.Context, name string, remoteInsta
 		defer func() {
 			span.SetAttributes(
 				attribute.String("cow_name", name),
-				attribute.Int64("uncompressed_bytes", size),
+				attribute.Int64("cow_size_bytes", size),
```

Comment:
Just to be extra explicit, maybe add a comment that it includes non-dirty and zero chunks?